components/Dropdown/Dropdown.less

```diff
@@ -36,7 +38,9 @@
     }
   }
   .dropdown-hidden {
-    display: none;
+    // display: none;
+    // opacity: 0;
+    left: -500px;
   }
 }
```

Comment:
- Noble, great job on the assignment. 
- You completed the MVP by implementing Dropdown functionality along with Tabs components. Not only that but You added animations to make whole experience more smooth.

- Your code is clean and git workflow is fluid as usual, great job 👍